src/backend/oracle-data-api/src/main/java/ca/bc/gov/open/jag/tco/oracledataapi/controller/v1_0/EmailHistoryController.java

```diff
@@ -60,7 +61,20 @@ public List<EmailHistory> getEmailHistoryByTicketNumber(
 	@PostMapping("/emailHistory")
 	public ResponseEntity<Long> insertEmailHistory(
 			@RequestBody EmailHistory emailHistory) {
-		logger.debug("POST /emailHistory called to save emailHistory: {}", StructuredArguments.fields(emailHistory));
-		return new ResponseEntity<Long>(emailHistoryService.insertEmailHistory(emailHistory), HttpStatus.OK);
+		logger.debug("POST /emailHistory called with: {}", StructuredArguments.fields(emailHistory));
+		// return bad request for invalid arguments
+		if (emailHistory.getOccamDisputeId() == null || emailHistory.getOccamDisputeId() == 0)
+			throw new IllegalArgumentException("Occam dispute id is zero or missing.");
+		else if (emailHistory.getHtmlContent() != null && emailHistory.getPlainTextContent() != null )
+			throw new IllegalArgumentException("Only one of plaintextcontent or htmlcontent can be supplied.");
+		else if (emailHistory.getSuccessfullySent() == null || emailHistory.getSuccessfullySent() == YesNo.UNKNOWN)
+			throw new IllegalArgumentException("parameter successfullySent has unknown value");
```

Comment:
check for all illegal arguments that ords endpoint checks for so we dont get a 500 exception